terraform/lambda/dynamodb.tf

```diff
@@ -44,3 +57,31 @@ resource "aws_iam_policy" "access-user-issued-credentials-table" {
     ]
   })
 }
+
+resource "aws_iam_policy" "access-auth-codes-table" {
```

Comment:
I think you're going to need a bit more terraform to attach this policy to the lambda execution role. You can see how I did it for the user-cred table here: https://github.com/alphagov/di-ipv-core-back/blob/main/terraform/modules/endpoint/iam.tf#L27

Using this pattern you can scope the perms to just the lambdas that need access to that table.

Reply:
Ah yeah good shout thanks. I've added the additional terraform resources that I think are required.